nni/algorithms/compression/v2/pytorch/pruning/tools/base.py

```diff
@@ -339,6 +342,8 @@ def __init__(self, pruner: Compressor, dim: Optional[Union[int, List[int]]] = No
             Example:
 
             The metric size is (12,), and block_sparse_size=[64], then the mask will expand to (768,) at first before expand with `dim`.
+        continuous_mask
+            Inherit the mask already in the wrapper if set True.
```

Comment:
what is this for?

Reply:
whether continuous depends on specific sparsity allocator, is it proper to put this argument in base sparsity allocator?

Reply:
In fact, each sparsity allocator can have `continuous_mask`, so I put this in base,